### PR TITLE
Fixed the issue of absense of vagrant user on xenial64

### PR DESCRIPTION
**Pull request by pankajkgarg**
A fix for absence of 'vagrant' user on ubuntu/xenial64  vm machine.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -338,6 +351,17 @@ Vagrant.configure(2) do |config|
     b.vm.provision "run tests", :type => :shell, :privileged => false, :inline => run_tests("centos6_64")
   end
 
+  config.vm.define "xenial64" do |b|
+    b.vm.box = "ubuntu/xenial64"
+    b.vm.provider :virtualbox do |v|
+      v.memory = 768
+    end
+    b.vm.provision "packages debianoid", :type => :shell, :inline => packages_debianoid
+    b.vm.provision "build env", :type => :shell, :privileged => false, :inline => build_sys_venv("trusty64")
```
Comment (ThomasWaldmann) [inline Vagrantfile]: "trusty" (that's what I meant in my last comment) - see also below.

**Comment by pankajkgarg** [comment]
Oh, my bad, completely missed that part. Its fixed now.

**Comment by pankajkgarg** [comment]
Just one question, going ahead should I base my future commits from master branch or 1.0-maint ?

**Comment by ThomasWaldmann** [comment]
@pankajkgarg it depends. if it is useful for both branches and is a fix or otherwise advised for 1.0-maint, it often should go there and then automatically gets merged into master also.

**Comment by ThomasWaldmann** [comment]
lgtm, merge?

**Comment by pankajkgarg** [comment]
yupp

**Comment by PlasmaPower** [comment]
I'm running into a lot of problems with the `xenial64` VM, it seems like a mess. For instance, the home folder of "ubuntu" is set to `/root` for whatever reason. Also, since `/etc/hostname` doesn't exist `sudo` complains every time it's executed.

**Comment by ThomasWaldmann** [comment]
@PlasmaPower maybe talk to the creator of the VM. If they would just create their VM as everybody else does, life would be easier. That also applies to using "vagrant" user, not creating a special case just by using "ubuntu".

**Comment by enkore** [comment]
The bugtracker according to ATLAS is https://bugs.launchpad.net/cloud-images/+filebug/+login

**Pull request merged**